OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -404,15 +404,15 @@ public static String getSdkVersionRaw() {
    }
 
    private static OSLogger logger = new OSLogWrapper();
-   private static FocusTimeController focusTimeController = new FocusTimeController(new OSFocusTimeProcessorFactory(), logger);
```

Comment:
Why changing focusTimeController to lazy init avoid the carry-over 🤔  ? FocusTimeController constructor isn't triggering a functionality and is reset under StaticResetHelper. Am I missing something?

Reply:
Understood, My only concern with the lazy getter is that this goes in the contrary direction with the dependency injection pattern we want to follow.